### ISSUE TITLE
[Translation] People Депутати 

### ISSUE THREAD

**Issue by henare** [OPORA/publicwhip#31]
(e.g. https://theyvoteforyou.org.au/people) 

This is the main template https://github.com/OPORA/publicwhip/blob/develop/app/views/members/index.html.haml

<table class="c23" cellpadding="0" cellspacing="0"><tbody><tr class="c12"><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c3">English Version</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c3">Ukrainian Version</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c3">Notes</span></p></td></tr><tr class="c12"><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">Representatives and Senators</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">Народні депутати України</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">People’s Deputies</span></p></td></tr><tr class="c12"><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">The people who vote on your behalf in Parliament</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">Народні представники, ті, хто голосують від твого імені у парламенті </span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5 c11"><span class="c2"></span></p></td></tr><tr class="c12"><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">All</span></p><p class="c0 c5"><span class="c2">Representatives</span></p><p class="c0 c5"><span class="c2">Senate</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">Всі</span></p><p class="c0 c5"><span class="c2">Мажоритарники</span></p><p class="c0 c5"><span class="c2">За партійними списками</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">All</span></p><p class="c0 c5"><span class="c2">Single-Mandate</span></p><p class="c0 c5"><span class="c2">Party Lists?</span></p></td></tr><tr class="c12"><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">Sorted by…</span></p><ul class="c28 lst-kix_tcj7fggjljlz-0 start"><li class="c0 c5 c26"><span class="c2">Name</span></li><li class="c0 c5 c26"><span class="c2">Electorate / State</span></li><li class="c0 c5 c26"><span class="c2">Party</span></li><li class="c0 c5 c26"><span class="c2">Rebellions</span></li><li class="c0 c5 c26"><span class="c2">Attendance</span></li></ul><p class="c0 c5"><span class="c2">+</span></p><ul class="c28 lst-kix_tcj7fggjljlz-0"><li class="c0 c5 c26"><span class="c2">Faction</span></li><li class="c0 c5 c26"><span class="c2">Way they were elected (Single-Mandate/Party List)</span></li></ul></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">Сортувати за ...</span></p><p class="c0 c5"><span class="c2">● Ім'ям</span></p><p class="c0 c5"><span>● </span><span class="c21 c24">&nbsp;</span><span class="c2">Областю</span></p><p class="c0 c5"><span class="c2">● Партією</span></p><p class="c0 c5"><span>● </span><span class="c2">Дотриманням фракційної дисципліни</span></p><p class="c0 c5"><span class="c2">● Відвідуваністю</span></p><p class="c0 c5"><span class="c2">+</span></p><p class="c0 c5"><span class="c2">● Фракцією</span></p><p class="c0 c5"><span class="c2">● Способом обрання (мажоритарники/за списками)</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">Sorted by…</span></p><ul class="c28 lst-kix_tcj7fggjljlz-0"><li class="c0 c5 c26"><span class="c2">Name</span></li><li class="c0 c5 c26"><span class="c2">Single-Mandate/Party List (?)</span></li><li class="c0 c5 c26"><span class="c2">Oblast</span></li><li class="c0 c5 c26"><span class="c2">Faction</span></li><li class="c0 c5 c26"><span class="c2">Rebellions</span></li><li class="c0 c5 c26"><span class="c2">Attendance</span></li></ul></td></tr><tr class="c12"><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">X Party Representative for State Y</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">&nbsp;представник фракції Х від області Y (партії Z)</span></p><p class="c0 c5 c11"><span class="c2"></span></p><p class="c0 c5"><span class="c2">[X Faction Deputy for Oblast Y (or party list Z)]</span></p><p class="c0 c5 c11"><span class="c2"></span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">X Faction Deputy for Oblast Y (or party list Z)</span></p></td></tr><tr class="c12"><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">Never rebels </span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">100% фракційна дисципліна</span></p><p class="c0 c5 c11"><span class="c2"></span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2 c15">[against faction, not party]</span></p></td></tr><tr class="c12"><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">Rebels X% of the time </span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span>Проти </span><span class="c2">фракції у X% випадків</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5 c11"><span class="c2"></span></p></td></tr><tr class="c12"><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">X% Attendance</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5"><span class="c2">&nbsp;X% Відвідуваність</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c0 c5 c11"><span class="c2"></span></p></td></tr></tbody></table>

**Comment by henare** [comment]
Some notes on the translation so far:

In our data we've set the "party" to the faction, since that's what we're more interested in. Therefore it doesn't make sense to add a new sort by party option. I've labelled the existing sort by party as "Фракцією" (faction).

"X Party Representative for State Y" was a bit of a mind bender. I _think_ I've done what it says above but it could be very likely it's wildly wrong. It was done in 4f6dfb93c330f59e4bd7cf7e60bc0c6ed71787cc. How does it look @mmitre @lisoffsky?
